Removal of older versions of `ecstatic` is causing `npm install` to fail

The growth chart app depends on `http-server` 0.9.0, which depends on `ecstatic` <= 2.0. When cloning and doing `npm install`:

```
npm install
npm ERR! code ETARGET
npm ERR! notarget No matching version found for ecstatic@^1.4.0
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
npm ERR! notarget
npm ERR! notarget It was specified as a dependency of 'http-server'
npm ERR! notarget
```

Changing the http-server version:

```json
{
  "name": "growthchart",
  "version": "0.1.3",
  "scripts": {
    "serve": "./node_modules/http-server/bin/http-server -p 9000 -c-1 .",
    "start": "npm run serve"
  },
  "dependencies": {
    "eslint": "^3.9.1",
    "fhirclient": "^0.1.9"
  },
  "devDependencies": {
    "http-server": "^0.10.0"
  }
}
```
appears to fix it.

--- a/package.json
+++ b/package.json
@@ -10,6 +10,6 @@
     "fhirclient": "^0.1.9"
   },
   "devDependencies": {
-    "http-server": "^0.9.0"
+    "http-server": "^0.10.0"
   }
 }

